### DISCUSSION TITLE
Email verification with code

### DISCUSSION THREAD

**Discussion by sparkmylife** [keycloak/keycloak#13421] [Q&A]
Hi All,

I am looking a way to verify email with code instead of an activation link. I can see the email template themes/src/main/resources/theme/base/email/html/email-verification-with-code.ftl which seems does this, but how to enable this feature?

Thanks
Neeraj

**Comment by chromtechio** [comment]
Hi Friend, if you know the answer please share!

**Comment by MahmoudH96** [comment]
I would also be interested if you find a way to do so
